iModelJsNodeAddon/JsInteropDgnDb.cpp

```diff
@@ -1386,3 +1386,56 @@ void JsInterop::WriteAffectedElementDependencyGraphToFile(DgnDbR db, Utf8StringC
     ElementDependency::Graph graph(db.Txns());
     graph.WriteAffectedGraphToFile(BeFileName(dotFileName.c_str(), true), changedIds, {});
     }
+
+void JsInterop::CreateAnnotationTextStyle(DgnDbR db, Napi::Object obj)
+    {
+        BeJsConst textStyle(obj);
+        auto name = textStyle[json_name()].asCString();
+        auto colorTypeId = textStyle[json_colorType()].asUInt();
+        auto colorValue = textStyle[json_colorValue()].asUInt();
+        auto fontId = textStyle[json_fontId()].asUInt64();
+        auto height = textStyle[json_height()].asDouble();
+        auto lineSpacingFactor = textStyle[json_lineSpacingFactor()].asDouble();
+        auto isBold = textStyle[json_isBold()].asBool();
+        auto isItalic = textStyle[json_isItalic()].asBool();
+        auto isUnderlined = textStyle[json_isUnderlined()].asBool();
+        auto widthFactor = textStyle[json_widthFactor()].asDouble();
+        auto description = textStyle[json_description()].asCString();
+
+        auto ats = AnnotationTextStyle::Create(db.GetDictionaryModel());

```

Comment:
@abeesh, do you think we'll need the ability to insert definition elements like text styles into specific models (vs. the global dictionary)?